www/application/models/JadwalDosen_model.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class JadwalDosen_model extends CI_Model {
+
```

Comment:
menggunakan konstanta di model memunculkan eror "undefined constants in www/index.php" ,dan benar begitu saya simpan konstantanya di index.php erornya hilang. Apakah tidak apa-apa bila saya menyimpan konstatnanya di index.php, bukan di model ?